src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -1737,3 +1737,65 @@ def test_managed_private_endpoints(self):
         self.cmd(
             'az synapse managed-private-endpoints show --workspace-name {workspace} --pe-name {name}',
             expect_failure=True)
+
+    #@record_only()
+    @ResourceGroupPreparer(name_prefix='synapse-cli', random_name_length=16)
+    #@StorageAccountPreparer(name_prefix='adlsgen2', length=16, location=location, hns=True, key='storage-account')
+    @StorageAccountPreparer(name_prefix='adlsgen2', length=16, location=location, key='storage-account')
+    def test_spark_job_definition(self):
+        self.kwargs.update({
+            'workspace': 'testsynapseworkspacepe',
+            'name': 'SparkAutoCreate1',
+            'spark-pool': 'testpool',
+            'spark-version': '2.4',
+            'location': 'eastus',
```

Comment:
Remove no use args, like `workspace` and `location`. Because you create a new workspace below, the value of `workspace` will be overwritten.